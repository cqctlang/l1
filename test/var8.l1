
define f(x,y,z){
	@local g, c;
	c = x;
	define g() {
		c++;
		x++;
		y += 1;
		z = z+1;
		return;
	}
	g();
	g();
	printf("%d %d %d %d\n", c, x, y, z);
}

f(1,1,1);

