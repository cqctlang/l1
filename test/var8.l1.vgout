entry: 0 params, 0 locals, 0 temps
f: 3 params, 2 locals, 0 temps
g: 0 params, 0 locals, 0 temps
3 3 3 3
