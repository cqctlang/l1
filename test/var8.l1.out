entry: var params, 1 locals, 0 temps
f: 3 params, 2 locals, 0 temps
g: 0 params, 1 locals, 0 temps
3 3 3 3
